src/server/express.ts

```diff
@@ -43,6 +44,7 @@ export async function startExpressServer({
   app.post(path, createMcpServer);
   app.get(path, methodNotAllowed);
   app.delete(path, methodNotAllowed);
+  app.post('/jwt', generateJwt);
```

Comment:
This endpoint is for demonstration purposes only. I'll remove it before merging.